docker/README.md

```diff
@@ -0,0 +1,148 @@
+# Docker and exercism.io
+
+Optionally, one can develop (and even deploy) exercism.io using a tool
+called Docker. The primary advantage of using it is that you don't have to
+install a ton of dependencies--like a ruby version manager, ruby, postgres,
+nodejs, and a bunch of Ruby gems--on your own system. The disadvantage,
+however, is that you may need to acquaint yourself with Docker, and you
+need a 64-bit machine.
+
+For a basic overview, see [What is Docker?][] on the Docker website.
```

Comment:
No, this is great. I thought you were using the ID version but with a missing ID:

```
[this is the anchor text][anchor]

[anchor]: https://google.com?q=anchor
```